scripts/chocolatey/build/tools/chocolateyinstall.ps1

```diff
@@ -11,118 +11,32 @@ This script installs the Lacework Command Line Interface.
 
 $ErrorActionPreference = "stop"
 
-Set-Variable GithubReleasesRootUrl -Option ReadOnly -value "https://github.com/lacework/go-sdk/releases" -Force
-Set-Variable PackageName -Option ReadOnly -value "lacework-cli-windows-amd64.exe.zip" -Force
-
-Function Get-File($url, $dst)
-{
-    Write-Host "Downloading $url"
-    try
-    {
-        [System.Net.ServicePointManager]::SecurityProtocol = [Enum]::ToObject([System.Net.SecurityProtocolType],3072)
-    }
-    catch
-    {
-        Write-Error "TLS 1.2 is not supported on this operating system. Upgrade or patch your Windows installation."
-    }
-    $wc = New-Object System.Net.WebClient
-    $wc.DownloadFile($url, $dst)
-}
-
-Function Get-WorkDir
-{
-    $parent = [System.IO.Path]::GetTempPath()
-    [string]$name = [System.Guid]::NewGuid()
-    New-Item -ItemType Directory -Path (Join-Path $parent $name)
-}
-
-Function Get-Archive()
-{
-    $url = $GithubReleasesRootUrl
-    $package_name = $PackageName
-
-    $lacework_cli_url = "$url/download/v${lacework_cli_version}/${package_name}"
-
-    $sha_url = "$lacework_cli_url.sha256sum"
-    $cli_dest = (Join-Path ($workdir) "lacework-cli.zip")
-    $sha_dest = (Join-Path ($workdir) "lacework-cli.zip.shasum256")
-
-    Get-File $lacework_cli_url $cli_dest
-    $result = @{ "zip" = $cli_dest }
-
-    try
-    {
-        Get-File $sha_url $sha_dest
-        $result["shasum"] = (Get-Content $sha_dest).Split()[0]
-    }
-    catch
-    {
-        Write-Warning "No shasum exists for $version. Skipping validation."
-    }
-    $result
-}
-
-function Get-SHA256Converter
-{
-    if ($PSVersionTable.PSEdition -eq 'Core')
-    {
-        [System.Security.Cryptography.SHA256]::Create()
-    }
-    else
-    {
-        New-Object -TypeName Security.Cryptography.SHA256Managed
-    }
+$version = '0.36.0'
```

Comment:
should this version string have the `v`? that is, `v0.36.0` 🤔

Reply:
that doesn't seem to be the convention with chocolatey.

Reply:
I don't know if that will confuse our users, but if there is no way around it, we will let it go